src/crepe/api/LoopManager.cpp

```diff
@@ -0,0 +1,60 @@
+
+#include "../system/RenderSystem.h"
+#include "../system/ScriptSystem.h"
+#include "../facade/SDLContext.h"
+
+#include "LoopManager.h"
+#include "LoopTimer.h"
+
+namespace crepe {
+
+void LoopManager::process_input() {
+    SDLContext::get_instance().handle_events(this->game_running);
```

Comment:
Ik heb liever dat je in de constructor van LoopManager een reference naar SDLContext, LoopTimer, RenderSystem, etc. opslaat zodat dit later makkelijker te refactoren is als SDLContext geen singleton meer mag zijn